src/tests/lodashTest/isObjectLike.test.js

```diff
@@ -0,0 +1,31 @@
+import "https://unpkg.com/mocha@5.2.0/mocha.js"; /* globals mocha */
+import "https://unpkg.com/chai@4.1.2/chai.js"; /* globals chai */
+
+import isObjectLike from "/common/modules/lodash/isObjectLike.js";
+
+describe("common lodash: isObjectLike", function () {
```

Comment:
That's a very good intent, but I doubt we need to test lodash modules again. They are teted by lodash already and may/are in the future even considered to be dropped: https://github.com/TinyWebEx/AddonTemplate/issues/7

But sure this was a good test to do,

Same goes for the other files. `BrowserCommunicationTypes` is a thing I would keep, the other ones are kinda redundant.